package.json

```diff
@@ -26,8 +26,8 @@
     "@babel/preset-env": "^7.4.5",
     "@babel/register": "^7.4.4",
     "@sendgrid/mail": "^6.4.0",
-    "babel-plugin-module-resolver": "^3.2.0",
     "babel-plugin-istanbul": "^5.1.4",
+    "babel-plugin-module-resolver": "^3.2.0",
```

Comment:
Well done, would you mind help the team to sort out package.json, All the babel should be dependencies and chai and likes should be in dev-dependencies.

Reply:
Ok